client/src/components/survey/SurveyPage4.jsx

```diff
@@ -0,0 +1,356 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { useBlockstack } from 'react-blockstack';
+import { useHistory } from 'react-router-dom';
+import { Typography, Slider, Button, Grid, withStyles } from '@material-ui/core';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import buttonsCss from '../../css/buttons';
+import actions from '../../redux/actions/actions';
+import BehavioralChart from '../behavior/chart/BehavioralChart';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    fontFamily: 'Nunito',
+    maxWidth: theme.breakpoints.values.md,
+    width: '92vw',
+    [theme.breakpoints.up('sm')]: {
+      width: '50%',
+    },
+  },
+  controlLabels: {
+    marginLeft: -20,
+  },
+  buttons: {
+    ...buttonsCss.buttons,
+    width: '20vw',
+    height: '10vh',
+    margin: '1em',
+  },
+  continueButton: {
+    ...buttonsCss.buttons,
+    marginTop: '1vh',
+    width: '50vw',
+    maxWidth: '180px',
+  },
+  dialog: {
+    background: '#7a9cf9',
+  },
+  dialogText: {
+    color: 'white',
+    margin: '0 auto',
+  },
+  selectedButton: {
+    ...buttonsCss.buttons,
+    margin: '0.5em',
+  },
+  button: {
+    ...buttonsCss.buttons,
+    margin: '0.5em',
+    background: 'rgba(255,255,255,0.5)',
+    backgroundColor: `linear-gradient(45deg, #4760ff, #82a4f8)`,
+    color: 'black',
+    '&:hover': {
+      ...buttonsCss.buttons,
+      backgroundColor: `linear-gradient(45deg, #4760ff, #82a4f8)`,
+    },
+  },
+}));
+
+const WellnessSlider = withStyles({
+  markLabel: {
+    fontSize: '1.17rem',
+  },
+})(Slider);
+
+const marks = [
+  {
+    value: 1,
+    label: 'Not at All',
+  },
+  {
+    value: 1.5,
+    label: 1.5,
+  },
+  {
+    value: 2,
+    label: 2,
+  },
+  {
+    value: 2.5,
+    label: 2.5,
+  },
+  {
+    value: 3,
+    label: 'Somewhat',
+  },
+  {
+    value: 3.5,
+    label: 3.5,
+  },
+  {
+    value: 4,
+    label: 4,
+  },
+  {
+    value: 4.5,
+    label: 4.5,
+  },
+  {
+    value: 5,
+    label: 'A lot',
+  },
+];
+
+const SurveyPage4 = props => {
+  const { setSurveyPage4, toSurveyPage3, survey, addObservation, clearSurvey } = props;
+  const classes = useStyles();
+  const { interest, sadness, sleep, energy, appetite } = props;
+  const history = useHistory();
+  const { userSession } = useBlockstack();
+  const [interestAnswer, setInterestAnswer] = useState(interest);
+
+  const [sadAnswer, setSadAnswer] = useState(sadness);
+
+  const [sleepAnswer, setSleepAnswer] = useState(sleep);
+
+  const [energyAnswer, setEnergyAnswer] = useState(energy);
+
+  const [appetiteAnswer, setAppetiteAnswer] = useState(appetite);
```

Comment:
A little pedantic, but since there's more things to change - if you could just remove the spaces between these that would be great. Just to keep things semi-consistent between files